### ISSUE TITLE
Failed to parse subquery on where clause

### ISSUE THREAD

**Issue by joe-re** [joe-re/sql-language-server#53]
```sql
select * from employee
where hire_date = (select max(hire_date) from employee);
```

**Comment by zjuyxy** [comment]
I still got the same error with 0.11.3 version in sqlite3.

**Comment by joe-re** [comment]
@zjuyxy Thanks for your report. I'll investigate it.

**Comment by joe-re** [comment]
@zjuyxy Released v0.11.4 to fix the problem. Can you try it again with it?

**Comment by zjuyxy** [comment]
0.11.4 works fine.
